packages/suite-desktop/package.json

```diff
@@ -167,6 +167,8 @@
         "next-transpile-modules": "^2.3.1",
         "react": "16.13.1",
         "react-dom": "16.13.1",
+        "react-markdown": "4.3.1",
```

Comment:
Not sure why the react-markdown dependency was added here. I guess it's not required by spectron?

Reply:
yeah, it looks suspicious and does not make sense. I ll check it.